tests/cypress/tests/e2e/footer.spec.js

```diff
@@ -17,15 +17,9 @@ describe("Footer integration tests", () => {
   });
 
   it("Footer accessibility statement link navigates to the right external URL", () => {
-    cy.contains(accessibilityStatementLinkText)
-      .invoke("attr", "href")
-      .should(
-        "eq",
-        "https://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/CMSNondiscriminationNotice"
-      );
-    cy.contains(accessibilityStatementLinkText)
-      .invoke("attr", "target")
-      .should("eq", "_blank");
+    cy.get(
+      'a[href="https://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/CMSNondiscriminationNotice"]'
+    ).contains(accessibilityStatementLinkText);
```

Comment:
Moved to a slightly better way of calling and erroring if the footer link isn't found.